### PR TITLE
feat(gatsby-source-drupal): filter api requests by languages 

### PR DESCRIPTION
**Pull request by kathmbeck**
## Description

This adds a plugin option to filter api requests to drupal by the current language code.

### CODE REVIEW

File: packages/gatsby-source-drupal/src/gatsby-node.ts [hunk 1]
```diff
@@ -22,6 +22,7 @@ import {
   isFileNode,
   imageCDNState,
 } from "./normalize"
+import { filter } from "lodash"
```
Comment (TylerBarnes) [inline packages/gatsby-source-drupal/src/gatsby-node.ts:25]: was this auto imported accidentally?

File: packages/gatsby-source-drupal/src/plugin-options.ts [hunk 2]
```diff
@@ -12,6 +12,7 @@ type Options = {
   languageConfig?: {
     enabledLanguages?: Array<string | RenamedLangCode>
     renamedEnabledLanguages?: Array<RenamedLangCode>
+    filterByLanguages?: boolean
```
Comment (TylerBarnes) [inline packages/gatsby-source-drupal/src/plugin-options.ts:15]: Looks decent! I think the only thing left is to update the readme

File: packages/gatsby-source-drupal/src/gatsby-node.ts (hunk 1 above)
Comment (kathmbeck) [reply packages/gatsby-source-drupal/src/gatsby-node.ts:25]: Ohh it must have been, good catch.

**Review by TylerBarnes — APPROVED** [review]

**Comment by TylerBarnes** [comment]
The failing test was unrelated so I admin merged

**Pull request merged**